docs/sink-configuration.md

```diff
@@ -78,6 +78,25 @@ The following options are available:
 * `tenant` - Hawkular-Metrics tenantId (default: `heapster`)
 * `labelToTenant` - Hawkular-Metrics uses given label's value as tenant value when storing data
 
+### Kafka
+This sink supports monitoring metrics and events.
+To use the kafka sink add the following flag:
+
+```
+--sink=kafka:<KAFKA_SERVER_URL>[?<OPTIONS>]
```

Comment:
Note that the format was changed weeks ago, the other sink configuration lines are indented with four spaces.